config/checkstyle.properties

```diff
@@ -0,0 +1,4 @@
+checkstyle.suppressions.file=config/suppressions.xml
+checkstyle.header.file=https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java.header
+checkstyle.regexp.header.file=https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/java_regexp.header
+checkstyle.importcontrol.file=config/import-control.xml
```

Comment:
Please add new lines to all files that don't end with one if it doesn't cause any issues.
I see atleast 4 files missing them.